addons/addon-base-raas/packages/base-raas-post-deployment/lib/plugins/steps-plugin.js

```diff
@@ -15,7 +15,7 @@
 
 const CreateUserRoles = require('../steps/create-user-roles');
 const InjectServiceEndpoint = require('../steps/inject-service-endpoint');
-const CreateCloudFrontInterceptor = require('../steps/create-cloudfront-interceptor');
```

Comment:
any feature missing in China regions? should keep it for global regions.

Reply:
Comment it because it is for lambda edge